CONTRIBUTING.md

```diff
@@ -182,3 +182,10 @@ public RealmQuery<E> equalTo(String fieldName, String fieldValue, boolean caseSe
 
 Above is based on the official guidelines from Oracle regarding Javadoc: http://www.oracle.com/technetwork/articles/java/index-137868.html
 
+### Branch Strategy
+
+* `releases` branch is where the hotfix versions released from. Releases from this branch will bump the last part of the version number.
```

Comment:
`releases` is where patch versions are released from. Releases from this branch will only bump the patch version, e.g. `1.0.0` to `1.0.1`